pedsnetdcc/tests/utils_test.py

```diff
@@ -178,3 +199,64 @@ def test_user_at_end(self):
         expected = {'search_path': None, 'host': 'ahost',
                     'dbname': 'adb', 'port': None, 'user': 'auser'}
         self.assertEqual(conn_info, expected)
+
+
+class SetLoggedTest(unittest.TestCase):
+
+    def setUp(self):
+        # Create a postgres database in a temp directory.
+        self.postgresql = Postgresql()
+        self.dburi = self.postgresql.url()
+        self.conn_str = make_conn_str(self.dburi)
+        self.model_version = '2.3.0'
+        self.engine = sqlalchemy.create_engine(self.dburi)
+        self.metadata = stock_metadata(self.model_version)
+
+    def tearDown(self):
+        # Destroy the postgres database.
+        self.postgresql.stop()
+
+    def test_set_logged(self):
+
+        self.metadata.tables['person'].create(self.engine)
+
+        stmt = Statement('ALTER TABLE person SET UNLOGGED')
+        stmt.execute(self.conn_str)
+        self.assertFalse(stmt.err)
+
+        set_logged(self.conn_str, self.model_version, tables=['person'])
+
+        stmts = StatementList()
+        stmts.append(Statement('SELECT pg_is_in_recovery'))
```

Comment:
Interesting, how does this test work?  Is there a link describing the technique?  I had also found this re: detected logged/unlogged: http://stackoverflow.com/questions/29899359/how-to-check-table-unlogged-with-postgresql